Messaging.md

```diff
@@ -165,10 +173,69 @@ This type contains signatures of the funding transaction and any necessary infor
 `witness` is the data for a witness element in a witness stack. An empty `witness_stack` is an error,
 as every input must be Segwit. Witness elements should *not* include their length as part of the witness data.
 
+### The `event_descriptor` Type
+
+This type contains information about the *exact* and fully specified outcomes  in an event for which an oracle plans on releasing a signature over.
+
+#### Version 0 `external_event_descriptor`
+
+1. type: 55300 (`external_event_descriptor_v0`)
+2. data:
+   * [`string`:`external_name`]
+
+`external_name` can refer to anything here and it is up to the oracle and user to agree on how to interpret it.
+
+#### Version 0 `enum_event_descriptor`
+
+1. type: 55302 (`enum_event_descriptor_v0`)
+2. data:
+   * [`u16`:`outcome_1_len`]
+   * [`utf8_string`:`outcome_1`]
+   * ...
+   * [`u16`:`outcome_n_len`]
+   * [`utf8_string`:`outcome_n`]
+
+This type of event descriptor is a simple enumeration where the value `n` is the number of outcomes in the event. `n` is not given because it can be found by parsing each outcome's length followed by the actual outcome until you has parsed the same number of bytes as the length field of the TLV.
+
+Each `outcome_i` corresponds to the pre-image of a possible outcome that the oracle could sign.
```

Comment:
I feel this sentence is useless, and "pre-image of a possible outcome" doesn't really make sense to me.

Reply:
Wanted to be clear that it isn't the hash like in the `contractInfo`